qw5q_platinum/platform.py

```diff
@@ -44,8 +44,7 @@ def create():
 
     # create qubit objects
     runcard = load_runcard(FOLDER)
-    kernels = Kernels.load(FOLDER)
```

Comment:
Instead of just removing the code that loads kernels, I would suggest to load the kernels conditionally based on the existence of the file, or replace the kernel file with an empty kernel file.

Reply:
Makes sense, now kernels are loaded only if the file exists: https://github.com/qiboteam/qibolab_platforms_qrc/pull/168/commits/413a54f02117f0c264e5d1ad44a19ecd31f5af50